rfps/rfp-consensus-visualization-tool.md

```diff
@@ -0,0 +1,123 @@
+# RFP: Consensus Visualization Tool
+
+**Name of Project:** `Consensus Visualization Tool`
+
+**Proposal Category:** `app-dev`
+
+## Project Description
+
+We are seeking proposals for an open source blockchain consensus visualization tool that demonstrates Filecoin's Expected Consensus process. There is currently a chain visualizer at http://user.kittyhawk.wtf:8000/ however it only shows the heaviest chain and does not include potential chain forks.
+
+Expected Consensus is different from other popular blockchain consensus algorithms and based on the heaviest weighted chain which tracks the most total storage power.
```

Comment:
EC is not different from other consensus algorithms because of the heaviest chain idea (almost all consensus algorithms have that). Or do you mean that it's different because we weight by storage power? Either way, can you reword this sentence to reduce ambiguity?

Reply:
The only other chain that has a notion of weight or weighted chain afaik is Algorand. So EC is different from Proof of Work and Proof of Stake. Other chains do not have a notion of heaviest but longest and most trustworthy. An earlier version of the Spec actually said that EC is based on weighted chains.

Reply:
Can you suggest a rewording? I've based this on my reading of the Spec.

Reply:
I will update to "Expected Consensus invokes leader election to select a miner from a weighted set of participants and ensures chain growth and convergence." which is directly from an existing doc.

Reply:
@pooja here's the revised version, let me know if you disagree with it.

Expected Consensus is different from other popular blockchain consensus algorithms like Proof of Work and somewhat different from Proof of Stake. It based on the heaviest weighted chain which tracks the most total storage power.

A miner's odds of being elected to produce a block in Expected Consensus is proportional to their storage power in the network, or proven storage in active use. Blocks are produced tracking proven storage in the network. Total proven storage in use is how chains are weighted. The network eventually converges around the heaviest chain.

Reply:
This looks great!